src/stores/AccountsStore.js

```diff
@@ -296,6 +296,8 @@ export default class AccountsStore extends Container<AccountsStoreState> {
 
 	async saveNewIdentity(seedPhrase, pin) {
 		const updatedIdentity = deepCopyIdentity(this.state.newIdentity);
+		//TODO encrypt seedPhrase with password in the future version,
+		// current encryption with only seedPhrase is compatible.
 		updatedIdentity.encryptedSeed = await encryptData(seedPhrase, pin);
```

Comment:
Currently we only encrypt `seedPhrase` in `encryptedSeed`, but will include `password` in the future